keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -200,7 +201,7 @@ Add ability to support drop-in configuration directory.
 
 #### Beta
```

Comment:
let's add the ability to query the effective configuration from the kubelet and tests for this as a graduation criteria.

Reply:
I'd also suggest to make it explicit that querying an effective config will be working for both - standard mode and standalone kubelet mode.

Reply:
Fixed